lua_protobuf/generator.py

```diff
@@ -999,7 +1012,7 @@ def file_header(file_descriptor):
     lines.extend(c_header_header(filename, package))
 
     for descriptor in file_descriptor.message_type:
-        lines.extend(message_header(package, descriptor))
+        lines += message_header_recursive(package, descriptor)
```

Comment:
Please change this back to `lines.extend()`, as I have a personal bias against the `+=` operator because it often doesn't do what you intended and can mask bugs.